packages/fxa-settings/scripts/test-ci.sh

```diff
@@ -1,5 +1,31 @@
 #!/bin/bash -ex
 
-yarn workspaces focus fxa-settings
-PUBLIC_URL=/ INLINE_RUNTIME_CHUNK=false CI=false SKIP_PREFLIGHT_CHECK=true yarn build
 CI=yes SKIP_PREFLIGHT_CHECK=true yarn test
+
+mkdir -p config
+
+cd ../../
+mkdir -p ~/.pm2/logs
+mkdir -p artifacts/tests
+yarn workspaces foreach \
+    --verbose \
+    --topological-dev \
+    --include browserid-verifier \
+    --include fxa-auth-db-mysql \
+    --include fxa-auth-server \
+    --include fxa-content-server \
+    --include fxa-profile-server \
+    --include fxa-react \
+    --include fxa-settings \
+    --include fxa-shared \
+    --include fxa-graphql-api \
+    run start > ~/.pm2/logs/startup.log
+npx pm2 ls
+# ensure email-service is ready
+_scripts/check-url.sh localhost:8001/__heartbeat__
```

Comment:
I ran `circleci local execute -c .circleci/local.yml --job test-settings-server` locally and it hung here. I'm assuming this is a local-only thing that I'm not doing. Just wanted to note.

Reply:
I've never been able to run `circleci local execute` so probably not the best person to answer 😬